mk/spksrc.cross-go-env.mk

```diff
@@ -1,8 +1,7 @@
 # Configuration for go build
 # 
 
-GENERIC_ARCHS = ARM7
```

Comment:
Be careful  with the `GENERIC_ARCHS` as I had found it had a direct impact on the build process where sub-arm7 targets gets built instead of the generic arm7.  Does your change keeps this or is it addressed differently?

Reply:
I cannot see any sideeffect by the removal of `GENERIC_ARCHS`.
IMHO it is obsolete since the introduction of generic armv7 arch (or at least since all-supported builds armv7 generic arch).

Reply:
Perhaps someone knows where this came from originally...  From the history it looks like this came from @ymartin59 , any clue on this, is this still a requirement?

Reply:
That was me that introduced it with commit 27445308dfa27b2892b1ddfb9af721b348ffbbdb 😄

Reply:
Even better.. :1st_place_medal: